src/email-sender.js

```diff
@@ -9,9 +9,9 @@ const nodemailer = require('nodemailer');
 exports.sendMessage = async function (errorMessage) {
   // Recipients list of users to send the email too
   // receipiants is commas separated (emailName@hotmail.com,emailName2@hotmail.com)
-  var receipiants = '';
+  const receipiants = '';
```

Comment:
Hey @RyanWils ! Something that I've noticed is that we have a recipients const, but we never have a way to update who the recipients are in the scope of this function. Maybe we can add a way to pass sendMessage the recipients in a array of strings?

Reply:
Hi Neil I was thinking about the same changes to make to the send message function, but i think that should be implemented down the road because it would be a hassle to create an array of string every time you want to call this function. We can implement an optional parameters down the road once we start using it.